conditional/config.py

```diff
@@ -31,5 +35,9 @@
     'post_logout_redirect_uris': [env.get("CONDITIONAL_OIDC_CLIENT_LOGOUT", "http://0.0.0.0:6969/logout")]
 }
 
+# Openshift secret
+SECRET_KEY = env.get("CONDITIONAL_SECRET_KEY", default=''.join(
+    random.SystemRandom().choice(string.ascii_uppercase + string.digits) for _ in range(64)))
```

Comment:
I think it would be better to have the run fail if `CONDITIONAL_SECRET_KEY` wasn't provided in the environment. By randomly generating this value on application start we are no longer able to have multiple instances of conditional running (under the same OIDC credentials). This is due to how pyoidc authentication relies on the `SECRET_KEY` being set in `app.config` and ties that in with session cookie data.

Reply:
So currently the build **will fail** if the secret key isn't provided. But it doesn't matter what it is. But yeah not being able to have multiple instances could be a problem, unless it's decided to give dev and prod different credentials.

Reply:
You should use the `secrets` library because security.